gtime/forecasting/simple_models.py

```diff
@@ -0,0 +1,354 @@
+import numpy as np
+import pandas as pd
+from sklearn.base import BaseEstimator, RegressorMixin
+from sklearn.utils.validation import check_is_fitted
+
+
+class NaiveModel(BaseEstimator, RegressorMixin):
+
+    """Naïve model, all predicted values are equal to the most recent available observation.
+
+    Examples
+    --------
+    >>> import pandas as pd
+    >>> import numpy as np
+    >>> from gtime.model_selection import horizon_shift, FeatureSplitter
+    >>> from gtime.forecasting import NaiveModel
+    >>> idx = pd.period_range(start='2011-01-01', end='2012-01-01')
+    >>> np.random.seed(1)
+    >>> df = pd.DataFrame(np.random.random((len(idx), 1)), index=idx, columns=['1'])
+    >>> y = horizon_shift(df, horizon=3)
+    >>> X_train, y_train, X_test, y_test = FeatureSplitter().transform(df, y)
+    >>> m = NaiveModel()
+    >>> m.fit(X_train, y_train).predict(X_test)
+                         y_1       y_2       y_3
+        2011-12-30  0.541559  0.541559  0.541559
+        2011-12-31  0.974740  0.974740  0.974740
+        2012-01-01  0.636604  0.636604  0.636604
+    """
+
+    def fit(self, X: pd.DataFrame, y=None):
+
+        """Fit the estimator.
+
+        Parameters
+        ----------
+        X : pd.DataFrame, shape (n_samples, n_features), train sample, required for compatibility, not used for a naive model.
+
+        y : None
+            Used to store the predict feature names and prediction horizon.
+
+        Returns
+        -------
+        self : NaiveModel
+            Returns self.
+        """
+
+        self._y_columns_ = y.columns
```

Comment:
For this and horizon, you can remove the underscore at the beginning

Reply:
fixed